workflow/scripts/osemosys_global/summary/constants.py

```diff
@@ -0,0 +1,35 @@
+"""Constants for summary statistics"""
+
+RENEWABLES = ["GEO", "HYD", "SPV", "CSP", "WAS", "WAV", "WON", "WOF"]
```

Comment:
BIO should probable be included in RENEWABLES here as this is currently also the case in other constant files (e.g. powerplant/constants.py).